examples/refrigerator-app/silabs/src/RefrigeratorUI.cpp

```diff
@@ -0,0 +1,303 @@
+}
+
+/**
+ * @brief Set the operating mode of the refrigerator.
+ * @param uint8_t mode Mode of the refrigerator (e.g., cooling, defrosting)
+ */
+void RefrigeratorUI::SetMode(uint8_t mode)
+{
+    mMode = mode;
+}
+
+/**
+ * @brief Draw the header of the UI, usually displaying the title or current status.
+ * @param GLIB_Context_t * pointer to the context for the GLIB library
+ */
+void RefrigeratorUI::DrawHeader(GLIB_Context_t * glibContext)
+{
+    // Draw Silabs Corner icon
+    GLIB_drawBitmap(glibContext, SILABS_ICON_POSITION_X, STATUS_ICON_LINE, SILABS_LOGO_WIDTH, SILABS_LOGO_HEIGHT, silabsLogo);
+    // Draw BLE Icon
+    GLIB_drawBitmap(glibContext, BLE_ICON_POSITION_X, STATUS_ICON_LINE, BLUETOOTH_ICON_SIZE, BLUETOOTH_ICON_SIZE, bleLogo);
+    // Draw WiFi/OpenThread Icon
+    GLIB_drawBitmap(glibContext, NETWORK_ICON_POSITION_X, STATUS_ICON_LINE, (UI_WIFI) ? WIFI_BITMAP_HEIGHT : THREAD_BITMAP_WIDTH,
+                    WIFI_BITMAP_HEIGHT, (UI_WIFI) ? wifiLogo : threadLogo);
+    // Draw Matter Icon
+    GLIB_drawBitmap(glibContext, MATTER_ICON_POSITION_X, STATUS_ICON_LINE, MATTER_LOGO_WIDTH, MATTER_LOGO_HEIGHT, matterLogoBitmap);
+#if SL_LCDCTRL_MUX
+    sl_wfx_host_pre_lcd_spi_transfer();
+#endif // SL_LCDCTRL_MUX
+    DMD_updateDisplay();
+#if SL_LCDCTRL_MUX
+    sl_wfx_host_post_lcd_spi_transfer();
+#endif // SL_LCDCTRL_MUX
+}
+
+/**
+ * @brief Draw the footer of the UI, which may display additional information such as mode.
+ * @param GLIB_Context_t * pointer to the context for the GLIB library
+ * @param bool autoMode Whether the refrigerator is in automatic mode
+ */
+void RefrigeratorUI::DrawFooter(GLIB_Context_t * glibContext, bool autoMode)
+{
+    switch (static_cast<RefrigeratorUI::SUPPORTED_MODES>(mMode))
+    {
+    case SUPPORTED_MODES::RAPID_COOL:
+        GLIB_drawStringOnLine(glibContext, "Mode : Rapid Cool", 11, GLIB_ALIGN_LEFT, 0, 0, true);
+        GLIB_drawBitmap(glibContext, HEATING_COOLING_X, HEATING_COOLING_Y, COOLING_WIDTH, COOLING_HEIGHT, heating_bits);
+        DrawSetPoint(glibContext, mCurrentTempCelsius, false);
+        break;
+    case SUPPORTED_MODES::RAPID_FREEZE:
+        GLIB_drawStringOnLine(glibContext, "Mode : Rapid Freeze", 11, GLIB_ALIGN_LEFT, 0, 0, true);
+        GLIB_drawBitmap(glibContext, HEATING_COOLING_X, HEATING_COOLING_Y, COOLING_WIDTH, COOLING_HEIGHT, cooling_bits);
+        DrawSetPoint(glibContext, mCurrentTempCelsius, false);
+        break;
+    case SUPPORTED_MODES::NORMAL:
+        GLIB_drawStringOnLine(glibContext, "Mode : Normal", 11, GLIB_ALIGN_LEFT, 0, 0, true);
+        GLIB_drawBitmap(glibContext, HEATING_COOLING_X, HEATING_COOLING_Y, COOLING_WIDTH, COOLING_HEIGHT, heating_cooling_bits);
+        DrawSetPoint(glibContext, mCurrentTempCelsius, false);
+        break;
+    case SUPPORTED_MODES::ENERGY_SAVE:
+        DrawSetPoint(glibContext, 0, false);
+        GLIB_drawStringOnLine(glibContext, "Mode : Energy save", 11, GLIB_ALIGN_LEFT, 0, 0, true);
```

Comment:
```suggestion
        GLIB_drawStringOnLine(glibContext, "Mode : Energy Save", 11, GLIB_ALIGN_LEFT, 0, 0, true);
```

aligning with the text casing of other modes